sdk/python/tests/test_online_retrieval.py

```diff
@@ -98,6 +98,7 @@ def test_online() -> None:
                 "customer_driver_combined:trips",
             ],
             entity_rows=[{"driver": 1, "customer": 5}, {"driver": 1, "customer": 5}],
+            full_feature_names=True,
```

Comment:
for tests that aren't testing full_feature_names, we should leave as default so no need to include this parameter in many of these cases and instead edit the expected output. Then directly test for full_feature_names behavior by having a test with it turned true

Reply:
Changed `full_feature_names` to default for all these tests and edited expected output. I do not test the full_feature_names feature here as it is tested thoroughly in the file `test_offline_online_store_consistency.py`.